packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -193,8 +193,8 @@ class _Elements {
 
   static void use(SvgParser parserState, bool warningsAsErrors) {
     final ParentNode? parent = parserState.currentGroup;
-    final String xlinkHref = parserState._currentAttributes.href!;
-    if (xlinkHref.isEmpty) {
+    final String? xlinkHref = parserState._currentAttributes.href;
+    if (xlinkHref == null || xlinkHref.isEmpty) {
```

Comment:
Separate patch?

Reply:
Yeah, this one is odd. This is required to compile all of the svgs in flutter_svg, specifically without this an empty `<use />` throws a compiler exception and I didn't think that was expected. Happy to land it separately though

Reply:
reverted